### ISSUE TITLE
32bit?

### ISSUE THREAD

**Issue by lwg2000** [Cobertos/unitypackage_extractor#4]
it doesn't run on win7 32bit any suggestion?

**Comment by Cobertos** [comment]
Hmm, I'll look into building it for 32 bit

**Comment by Cobertos** [comment]
Looked into this any seemingly it should be working on 32 bit.

Are you running the `.exe` or from Python? Are you using Python 3.6+ and the correct bitness of Python?
Are you getting any specific error or behavior?

I built it with both the 32 bit and 64 bit version of Python and both came out with 32 bit .dll's and .exe's...

```bash
cobertos@DESKTOP-1V2V7M0 MINGW64 ~/Workspace/projects/unitypackage_extractor2/unitypackage_extractor
$ find . -type f | xargs file
./base_library.zip:       Zip archive data, at least v2.0 to extract
./extractor.exe:          PE32 executable (console) Intel 80386, for MS Windows
./extractor.exe.manifest: XML 1.0 document, ASCII text, with CRLF line terminators
./pyexpat.pyd:            PE32 executable (DLL) (GUI) Intel 80386, for MS Windows
./python36.dll:           PE32 executable (DLL) (GUI) Intel 80386, for MS Windows
./select.pyd:             PE32 executable (DLL) (GUI) Intel 80386, for MS Windows
./unicodedata.pyd:        PE32 executable (DLL) (GUI) Intel 80386, for MS Windows
./VCRUNTIME140.dll:       PE32 executable (DLL) (console) Intel 80386, for MS Windows
./_bz2.pyd:               PE32 executable (DLL) (GUI) Intel 80386, for MS Windows
./_hashlib.pyd:           PE32 executable (DLL) (GUI) Intel 80386, for MS Windows
./_lzma.pyd:              PE32 executable (DLL) (GUI) Intel 80386, for MS Windows
```

**Comment by Cobertos** [comment]
My bad, that was the output of the newly built .zip, I'm definitely shipping x64 binaries currently.
I will update.

**Comment by Cobertos** [comment]
Fixed in 0.5.0
